converter.go

```diff
@@ -134,21 +151,21 @@ func valueToString(v driver.Value, tsmode snowflakeType) (*string, error) {
 		s := v1.String()
 		return &s, nil
 	case reflect.Struct:
-		if tm, ok := v.(time.Time); ok {
-			switch tsmode {
-			case dateType:
+		if tm, ok := v.(time.Time); ok && dataType != nil {
+			switch {
+			case dataType.Equals(DataTypeDate):
 				_, offset := tm.Zone()
 				tm = tm.Add(time.Second * time.Duration(offset))
-				s := fmt.Sprintf("%d", tm.Unix()*1000)
+				s := strconv.FormatInt(tm.Unix()*1000, 10)
```

Comment:
Weird that they changed this `fmt.Sprintf("%d")` to `strconv.FormatInt` but not any of the other ones below...

Reply:
Yeah... there are definitely some interesting changes on their end